api/images.go

```diff
@@ -82,10 +83,16 @@ func GetDeploymentImages(c *gin.Context) {
 		return
 	}
 
-	selector := fmt.Sprintf("release=%s", release)
+	selector := fmt.Sprintf("%s=%s", helm.HelmReleaseNameLabel, release)
 	log.Infof("Label selector: %s", selector)
 
 	imageList, err := listAllImages(client, selector)
+	if err == nil && len(imageList) == 0 {
```

Comment:
Move this after the error checking

Reply:
if this is moved to after the error checking than the error checking must be repeated for the second `listAllImages` call.

Reply:
I don't mind, also, that's an entirely different issue (how errors are handled in API handlers).

"Better clean than clever" This code is unreadable.